### DISCUSSION TITLE
posts/Go/go-kit-tutorial-05/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#293] [Announcements]
https://www.liwenzhou.com/posts/Go/go-kit-tutorial-05/ 

 李文周的Blog提供免费的全套Go语言学习教程。本文主要介绍了go-kit的教程的第五篇，介绍了如何使用go-kit作为RPC客户端调用其他微服务。

**Comment by shuqingzai** [comment]
应该教教 [github.com/pkg/errors​](https://github.com/pkg/errors) 的使用？？错误堆栈信息保留对于业务开发应该十分重要
